lightning/src/ln/channelmonitor.rs

```diff
@@ -666,6 +675,12 @@ pub struct ChannelMonitor<ChanSigner: ChannelKeys> {
 	// actions when we receive a block with given height. Actions depend on OnchainEvent type.
 	onchain_events_waiting_threshold_conf: HashMap<u32, Vec<OnchainEvent>>,
 
+	// If we get serialized out and re-read, we need to make sure that the chain monitoring
+	// interface knows about the TXOs that we want to be notified of spends of. We could probably
+	// be smart and derive them from the above storage fields, but its much simpler and more
+	// Obviously Correct (tm) if we just keep track of them explicitly.
+	watch_outputs: HashMap<Sha256dHash, Vec<Script>>,
```

Comment:
nit: `watch_outputs` sounds more like a boolean. `watched_` or `monitored_` might be more explicit

Reply:
Hmm, I dont want to imply that they *are* watched, only that they *must be* watched. Went with outputs_to_watch.